### ISSUE TITLE
config.api_key should not be mandatory for installation

### ISSUE THREAD

**Issue by wvteijlingen** [jorttbv/bunq-client#10]
The API documentation from Bunq doesn't mention anything about the API key being required when creating an installation.

However, the [Configuration class raises an error when config.api_key is nil](https://github.com/jorttbv/bunq-client/blob/57de35f1efdb203637a6355971b63c8cd405d545/lib/bunq/client.rb#L44). This means it is impossible to create an installation without having an API key.

I've looked through the codebase and I think we can solve this as follows:
1. I see there is already a nil check when creating a `SessionServer` [here](https://github.com/jorttbv/bunq-client/blob/master/lib/bunq/session_servers.rb#L13).
2. Add the same check when creating a `DeviceServer` [here](https://github.com/jorttbv/bunq-client/blob/master/lib/bunq/device_servers.rb#L16).
3. Remove the check in the Configuration class.

**Comment by lvonk** [comment]
Yes that is true. Is there a reason why you want to split this other than it is not required? 
Because I rather have a single check that ensures the config has all it needs than needing to implement the check on multiple locations. To do anything useful with the api we need the api key at some point.

**Comment by wvteijlingen** [comment]
I have a webserver that acts as a 'proxy' to the Bunq API. So multiple clients, each with an unique API key, use my webserver to connect to Bunq.

When setting up the webserver, I need to run the installation once to send the public key to Bunq. And since the clients supply the API key to my server per request, I don't have a fixed API key for the installation.

**Comment by lvonk** [comment]
Ok, is see. Created https://github.com/jorttbv/bunq-client/pull/11.
